src/Features/VisualBasic/Portable/FileHeaders/VisualBasicFileHeaderHelper.vb

```diff
@@ -0,0 +1,32 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.FileHeaders
+Imports Microsoft.CodeAnalysis.VisualBasic.LanguageServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.FileHeaders
+    Friend NotInheritable Class VisualBasicFileHeaderHelper
+        Inherits AbstractFileHeaderHelper
+
+        Public Shared ReadOnly Instance As VisualBasicFileHeaderHelper = New VisualBasicFileHeaderHelper()
+
+        Private Sub New()
+            MyBase.New(VisualBasicSyntaxKinds.Instance)
+        End Sub
+
+        Public Overrides ReadOnly Property CommentPrefix As String
+            Get
+                Return "'"
+            End Get
+        End Property
+
+        Protected Overrides Function GetTextContextOfComment(commentTrivia As SyntaxTrivia) As String
+            If Not commentTrivia.IsKind(SyntaxKind.CommentTrivia) Then
+                Throw ExceptionUtilities.UnexpectedValue(commentTrivia.Kind())
+            End If
+
+            Return commentTrivia.ToFullString().Substring(1)
```

Comment:
any thought on if spans would be beneficial here to allow us to easily refer to sections of strings in a non-allocating fashion?

Reply:
`Span<T>` isn't usable in Visual Basic, but I used `Memory<T>` to avoid some of the allocations.